frankenstein/server.go

```diff
@@ -59,6 +59,11 @@ func AppenderHandler(appender storage.SampleAppender) http.Handler {
 					Value:     model.SampleValue(s.GetValue()),
 					Timestamp: model.Time(s.GetTimestampMs()),
 				})
+				if err != nil {
```

Comment:
Whoops, I accidentally deleted this error check again before PRing - thanks for readding.